components/Calendar.js

```diff
@@ -2,22 +2,36 @@ import React from "react";
 import Filter from "./Filter";
 import { Row, Col } from "react-bootstrap";
 
+const filterMapping = [
+	{
+		topic: "Meetings",
+		color: "#82aaff",
+	},
+	{
+		topic: "Career",
+		color: "#c792ea",
+	},
+	{
+		topic: "Socials",
+		color: "#ff5370",
+	},
+	{
+		topic: "Workshops",
+		color: "#ffd700",
+	},
+];
+
 const Calendar = () => {
 	return (
 		<div>
```

Comment:
Change this to `<>` It is a React.Fragment and reduces the overall code size since we aren't doing anything special with the div except for holding element